src/test/scala/beam/sflight/SfLightRouterSpec.scala

```diff
@@ -72,10 +72,10 @@ class SfLightRouterSpec extends AbstractSfLightSpec("SfLightRouterSpec") with In
       assert(response.itineraries.exists(_.tripClassifier == WALK_TRANSIT))
       val transitOption = response.itineraries.find(_.tripClassifier == WALK_TRANSIT).get
       assertMakesSense(transitOption.toBeamTrip)
-      assert(transitOption.totalTravelTimeInSecs == 1116)
+      assert(transitOption.totalTravelTimeInSecs == 1117)
```

Comment:
This is seemingly from a change in the NetworkReader. I did not chase it the whole way, but found that the states changed:

`State{vertex=34422, weight=82, backEdge=93357, durationSeconds=49, distance=62266, idx=4}`

versus 

`State{vertex=34421, weight=80, backEdge=93355, durationSeconds=48, distance=62198, idx=4}`

This comes from the TransportNetwork as loaded via the NetworkReader code.